.github/workflows/k6-action-image-release.yml

```diff
@@ -0,0 +1,69 @@
+name: Release K6 Action Image
+
+on:
+  push:
+    tags:
+      - "k6-action-image-*"
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get variables from tags
+        id: vars
+        run: |
+          tag=${GITHUB_REF/refs\/tags\/k6-action-/}
```

Comment:
does not match trigger tag `k6-action-image-*` vs `k6-action-*`

Reply:
I caught that as well. Fixed already.